content/stratis-release-notes-3.8.1.md

```diff
@@ -0,0 +1,58 @@
++++
+title = "Stratis 3.8.1 Release Notes"
+date = 2025-05-13
+weight = 41
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+Stratis 3.8.1, which consists of `stratisd 3.8.1` and `stratis-cli 3.8.1`
+is a bug fix release which addresses the following problems.
+
+For `stratis-cli`, the release fixes a bug where a user would be unable to
+start an encrypted pool previously created with any Stratis release less
+than 3.8.0.
+
+`stratis-cli 3.8.1` also contains better organized help text and displays 
+more complete information in the pool detail view.
+
+For `stratisd`, the release makes improvements to the Stratis support for
+mounting filesystems at boot. It introduces a new systemd unit file,
+`stratis-fstab-setup-with-network@.service`, which should be used when a
+filesystem's pool requires unlocking with the network present, as is the
+case when a pool is encrypted using NBDE (network-bound disk encryption).
+The fstab entry for the filesystem must include the `_netdev` option if this
+systemd unit file is used.
+
+An example fstab entry for a filesystem on a pool that is encrypted using
+NBDE should look something like this:
+
+`/dev/stratis/<POOL_NAME>/<FILESYSTEM_NAME> <MOUNTPOINT> xfs defaults,_netdev,x-systemd.requires=stratis-fstab-setup-with-network@<POOL_UUID>.service,x-systemd.after=stratis-fstab-setup-with-network@<POOL_UUID>.service 0 2`
+
+The `_netdev` option ensures that systemd delays attempting to mount the
+filesystem until after the network is up. The
+`stratis-fstab-setup-with-network@.service` unit itself will not run until
+the network is available. If the `_netdev` option is omitted, systemd will
+calculate a cyclic dependency between the unit generated by systemd from
+the fstab entry and the `stratis-fstab-setup-with-network@.service` unit
```

Comment:
This is not entirely correct either. I think it's better to just state `cyclic dependency` and leave it at that. The cyclic dependency is actually a long series of dependencies due to the unit file generated by the fstab generator dependending on both the network and local-fs.target. The `_netdev` option removes the dependency on local-fs.target and thus resolves the long chain of circular dependencies.

Reply:
Ok. I think the newest revision should be entirely correct.